search/client.go

```diff
@@ -59,15 +63,12 @@ func (c *client) Query(q Query) (*Response, error) {
 	}
 	buf := bytes.NewReader(marshalledQuery)
 
-	req, err := http.NewRequest("POST", c.endpoint, buf)
-	if err != nil {
-		return nil, err
+	headers := map[string]string{
+		"Content-Type": "application/json",
+		"Accepts":      "application/json",
```

Comment:
Same for "Content-Type" and "Accepts". You could check if there are already constants for those in Go